reoptjl/test/test_http_endpoints.py

```diff
@@ -181,27 +181,54 @@ def test_simulated_load(self):
         v2_response = json.loads(resp.content)   
         assert("Error" in v2_response.keys())
 
-    def test_emissions_profile_endpoint(self):
-        # Call to the django view endpoint dev/emissions_profile which calls the http.jl endpoint
+    def test_avert_emissions_profile_endpoint(self):
+        # Call to the django view endpoint dev/avert_emissions_profile which calls the http.jl endpoint
```

Comment:
If we add in the functionality of emissions decrease defaults, we can test that within this suite.  (I see that we have FUEL_DEFAULTS in two places - the API and REopt.jl - and perhaps we adopt that stance for now and just add a global dictionary.)

Reply:
@zolanaj  I think I'm not following as to why we would test the defaults in this particular place? This test is specifically for the avert (year 1) emissions profile endpoint, which doesn't return the annual percent decrease values.

Reply:
That's true, the defaults test you added works so this isn't necessary.  Thank you!